fluentlenium-kotest/src/test/kotlin/org/fluentlenium/adapter/kotest/KoTestConfig.kt

```diff
@@ -0,0 +1,20 @@
+package org.fluentlenium.adapter.kotest
+
+import io.github.bonigarcia.wdm.WebDriverManager
+import io.kotest.core.config.AbstractProjectConfig
+import io.kotest.core.listeners.Listener
+import io.kotest.core.listeners.TestListener
+import io.kotest.core.spec.Spec
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+class KoTestConfig : AbstractProjectConfig() {
+    override fun listeners(): List<Listener> =
+        listOf(object : TestListener {
+            override suspend fun beforeSpec(spec: Spec) {
+                withContext(Dispatchers.IO) {
+                    WebDriverManager.chromedriver().setup()
```

Comment:
Does setup produce a new driver every time its run?

Reply:
This does not create a WebDriver, its just properly configures everything so a specific WebDriver can be used by Selenium: https://github.com/bonigarcia/webdrivermanager#basic-usage

using this callback it would be invoked multiple times right? is there a better suited callback for that?
I don't think its problematic to run it multiple times though

Reply:
Yeah I think you'd want to use `beforeProject` probably:

```kotlin
object TestProjectListener : ProjectListener {
   override suspend fun beforeProject() {
      //here
   }
}
```